src/section-exponent-rules-and-scientific-notation.ptx

```diff
@@ -37,200 +37,40 @@
-                        </row>
-                        <row>
-                            <cell><m>(-10)^3</m></cell>
-                            <cell><m>-10</m></cell>
-                            <cell><m>3</m></cell>
-                            <cell><m>(-10)(-10)(-10)</m></cell>
-                            <cell><m>-1000</m></cell>
-                        </row>
-                        <row>
-                            <cell><m>(-1)^{4}</m></cell>
-                            <cell><m>-1</m></cell>
-                            <cell><m>4</m></cell>
-                            <cell><m>(-1)(-1)(-1)(-1)</m></cell>
-                            <cell><m>1</m></cell>
-                        </row>
-                        <row bottom="major">
-                            <cell><m>10^{5}</m></cell>
-                            <cell><m>10</m></cell>
-                            <cell><m>5</m></cell>
-                            <cell><m>(10)(10)(10)(10)(10)</m></cell>
-                            <cell><m>100,000</m></cell>
-                        </row>
-                    </tabular>
-                </table>
-            </solution>
-        </exercise>
-
-        <p>As we continue our study of algebra, we'll encounter more and more expressions and equations with <term>exponents</term>, often containing unknown, variable values. The exponent rules are a quick list that reminds us of how exponents work in a few specific situations. Let's work through <em>why</em> these rules work.</p>
-    </subsection> -->
-
-
     <subsection>
         <title>Products and Exponents</title>
-        <p>In the section <xref ref="section-introduction-to-exponent-rules" autoname="title"/>, we introduced three basic rules involving products and exponents.  We'll begin with a brief recap and explanation of these three exponent rules for multiplication.</p>
-
-        <paragraphs>
-            <title>Product Rule</title>
-            <p>When you're multiplying two expressions that have the same base, you can simplify the product by adding the exponents. <me>
-            x^m \cdot x^n = x^{m+n}</me></p>
-        </paragraphs>
-
-
-        <paragraphs>
-            <title>Power to a Power Rule</title>
-            <p>When you have a base raised to an exponent and that expression is raised to another exponent, you multiply the exponents. <me>\left(x^m\right)^n = x^{m \cdot n}</me></p>
-        </paragraphs>
-
-        <paragraphs>
-            <title>Product to a Power Rule</title>
-            <p>When you have a product raised to an exponent, you apply the exponent to each factor in the product. <me>\left(x\cdot y\right)^n = x^{n}\cdot y^{n}</me></p>   
+        <p>In the section <xref ref="section-introduction-to-exponent-rules">Section</xref>, we introduced three basic rules involving products and exponents.  We'll begin with a brief recap and explanation of these three exponent rules for multiplication.</p>
```

Comment:
While this is still open, want to change `In the section <xref ref="section-introduction-to-exponent-rules">Section</xref>` to `In <xref ref="section-introduction-to-exponent-rules">Section</xref>`?